docs/scripting.md

```diff
@@ -9,47 +9,45 @@ Hubot out of the box doesn't do too much, but it is an extensible, scriptable ro
 
 There are three main sources to load scripts from:
 
+* scripts loaded from external __npm packages__ and specified in `external-scripts.json`
 * all scripts __bundled__ with your hubot installation under `scripts/` dir
 * __community scripts__ specified in `hubot-scripts.json` and shipped in the `hubot-scripts` npm package
-* scripts loaded from external __npm packages__ and specified in `external-scripts.json`
 
-### Community Scripts
+### NPM Packages
 
-To use community scripts, place the name of the script in the `hubot-scripts.json` file. For example:
+There are hundreds of scripts written and maintained by the community. Find them by searching the [NPM registry](https://www.npmjs.com/browse/keyword/hubot-scripts) for `hubot-scripts <your-search-term>`. For example:
 
-```coffeescript
-["redis-brain.coffee", "shipit.coffee", "whatis.coffee", "<new-script-name>.coffee"]
+```
+$ npm search hubot-scripts github
+NAME                  DESCRIPTION
+hubot-deployer        Giving Hubot the ability to deploy GitHub repos to PaaS providers hubot hubot-scripts hubot-gith
+hubot-gh-release-pr   A hubot script to create GitHub's PR for release
+hubot-github          Giving Hubot the ability to be a vital member of your github organization
+…
 ```
 
-(Please check the [script catalog](http://hubot-script-catalog.herokuapp.com) and the [hubot-scripts organization](https://github.com/hubot-scripts) for scripts carefully crafted for you by lots of nice folks)
-
-### NPM Packages
-
-Another way is to install scripts via an npm package (you can check some of them [here](https://npmjs.org/search?q=hubot)).
+To use a script from an NPM package:
 
-To load those scripts to your hubot installation, you need to place them in the `external-scripts.json` file after adding the required npm packages to the `package.json` dependency section.
+1. Add the package to the list of `dependencies` into your `package.json`.
```

Comment:
It's better to tell people to use `npm install --save <package>`. This automatically adds it to the package.json so it saves having to know how package.json works. It also adds a specific version (ie the latest), so other developers will get a consistent version.